cairo-contracts/scripts/compile_all_cairo1.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+
+######## USAGE ########
```

Comment:
Can you add some doc here, explaining what this script does and how, what it will install etc

Reply:
Done, tell me if I forgot something

Reply:
@tdelabro